examples/temporal_graph_learning/README.md

```diff
@@ -0,0 +1,40 @@
+# Customized TGAT model for snapshot-based temporal graph learning
+
+This folder contains python codes that demonstrate how to leverage the GraphStorm Framework (GSF) for production level snapshot-based temporal graph data.
+This example uses the TGAT as GNN model and the MAG graph data.
```

Comment:
i thought you wanted to use GDELT.

Reply:
For this example, I am using a modified version of MAG from https://github.com/amazon-science/tgl.